tests/unit/allowable-actions-mixin.spec.ts

```diff
@@ -15,6 +15,13 @@ const vuetify = new Vuetify({})
 setActivePinia(createPinia())
 const businessStore = useBusinessStore()
 
+// mock the entire module
+// it's the only way to override any exported function
+jest.mock('@/utils/feature-flags', () => {
```

Comment:
Oh OK. So, this'll provide a way to actually use the GetFeatureFlag function in this test. I'll be keeping this in mind.

Reply:
The old code worked fine before the package updates... 🤷‍♂️